iam/api-client/src/main/java/iam/snippets/Quickstart.java

```diff
@@ -13,48 +13,161 @@
  * limitations under the License.
  */
 
-// [START iam_quickstart]
-
 package iam.snippets;
 
+// [START iam_quickstart]
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.json.jackson2.JacksonFactory;
-import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.cloudresourcemanager.CloudResourceManager;
+import com.google.api.services.cloudresourcemanager.model.Binding;
+import com.google.api.services.cloudresourcemanager.model.GetIamPolicyRequest;
+import com.google.api.services.cloudresourcemanager.model.Policy;
+import com.google.api.services.cloudresourcemanager.model.SetIamPolicyRequest;
 import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.ListRolesResponse;
-import com.google.api.services.iam.v1.model.Role;
 import com.google.auth.http.HttpCredentialsAdapter;
 import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
 import java.util.Collections;
 import java.util.List;
 
 public class Quickstart {
 
-  public static void main(String[] args) throws Exception {
-    // Get credentials
+  public static void main(String[] args) {
+    // TODO: Replace with your project ID.
+    String projectId = "your-project";
+    // TODO: Replace with the ID of your member in the form "user:member@example.com"
+    String member = "your-member";
+    // The role to be granted.
+    String role = "roles/logging.logWriter";
+
+    // Initializes the Cloud Resource Manager service.
+    CloudResourceManager crmService = null;
+    try {
+      crmService = initializeService();
+    } catch (IOException | GeneralSecurityException e) {
+      System.out.println("Unable to initialize service: \n" + e.toString());
+    }
+
+    // Grants your member the "Log writer" role for your project.
+    addBinding(crmService, projectId, member, role);
+
+    // Get the project's policy and print all members with the "Log Writer" role
+    Policy policy = getPolicy(crmService, projectId);
+    Binding binding = null;
+    List<Binding> bindings = policy.getBindings();
+    for (Binding b : bindings) {
+      if (b.getRole().equals(role)) {
+        binding = b;
+        break;
+      }
+    }
+    System.out.println("Role: " + binding.getRole());
+    System.out.print("Members: ");
+    for (String m : binding.getMembers()) {
+      System.out.print("[" + m + "] ");
+    }
```

Comment:
`member` insead of `m`

Reply:
I already defined "member" above, so I get an error if I try to use "member" here.

Reply:
k